docs/user_manual/type_data.md

```diff
@@ -133,6 +133,25 @@ deleted if the option has not been used by any existing samples.
 {% include userman-toplink.md %}
 
 
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Tissue Piece Type" %}
```

Comment:
Need to add heading to the table of contents (in `/docs/_posts/`)

Reply:
Added.